addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5247,7 +5247,7 @@ msgstr ""
 #: xbmc\video\dialogs\GUIDialogAudioSubtitleSettings.cpp
 #: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
 msgctxt "#12376"
-msgid "Set as default for all videos"
+msgid "Set as default for all files"
```

Comment:
Sure, but it is for all files and if you dont put that here then a user may not realise it and save some settings that may break some other settings for other files and complain his settings for everything are now broken.

In a ideal world that would be enough but this is a once for all job and also works on Video and AudioOSD as well As DSPOSD

If you ask me this label should come with a huge disclaimer 50 page long.